modules/rostests/apitests/shell32/ItemIDList.cpp

```diff
@@ -210,6 +220,29 @@ START_TEST(PIDL)
     else
         skip("?\n");
     ILFree(pidl);
+
+
+    CComPtr<IShellFolder> pInternet;
+    HRESULT hr = SHCoCreateInstance(NULL, &CLSID_Internet, NULL, IID_PPV_ARG(IShellFolder, &pInternet));
+    if (SUCCEEDED(hr))
+    {
+        PCWSTR pszUrl = L"http://example.com/page?query&foo=bar";
+        PIDLIST_RELATIVE pidlUrl;
+        hr = pInternet->ParseDisplayName(NULL, NULL, const_cast<PWSTR>(pszUrl), NULL, &pidlUrl, NULL);
+        ok_long(hr, S_OK);
+        if (hr == S_OK)
+        {
+            ok_int(pidlUrl->mkid.abID[0], 0x61);
```

Comment:
For the future, it may be of interest to have the pidl id defines into a undocshell sdk header, so that they could be used also in the tests.
```suggestion
            ok_int(pidlUrl->mkid.abID[0], 0x61); // PT_INTERNET_URL
```